index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8">
   <title>Fork Me! FCC: Test Suite Template</title>
   <link rel="stylesheet" href="css/style.css">
-</head>
+</head> 
 
 <body>
   <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- [ ] You could try changing the feature class div with List items to a ul tag. :)

Reply:
- [ ] Let's try to apply both of my last suggestions in this comment too.

Reply:
What if you try changing the feature class div with List items to a ul tag? For the correct functioning of an li tag, we always need to nest it inside a ul tag as this is the correct CSS convention. 🙂